### PR TITLE
Zq/update abs with aclnn

### PR DESCRIPTION
**Pull request by NeosZhang**
## Motivation and Context

update diopiAbs and diopiAbsInp with aclnn

## Description

update diopiAbs and diopiAbsInp with aclnn

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by yangbofun — APPROVED** [review]

File: impl/ascend/functions/abs.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace impl {
+namespace ascend {
+diopiError_t diopiAbs(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t input) {
+    DIOPI_ASCEND_CALL_ACLNN(aclnnAbs, ctx, input, out);
+    return diopiSuccess;
+}
+
+diopiError_t diopiAbsInp(diopiContextHandle_t ctx, diopiTensorHandle_t input) {
+    diopiAbs(ctx, input, input);
```
Comment (yangbofun) [inline impl/ascend/functions/abs.cpp:18]: 华为有absinp接口么？

Comment (jingguo-st) [reply impl/ascend/functions/abs.cpp:18]: 不这样调，直接调华为的aclnnInplaceAbs

Comment (NeosZhang) [reply impl/ascend/functions/abs.cpp:18]: abs没有inp的版本
image

Comment (NeosZhang) [reply impl/ascend/functions/abs.cpp]: done

**Pull request merged**